test-utils/src/main/java/apoc/util/Neo4jContainerExtension.java

```diff
@@ -125,13 +117,6 @@ public Neo4jContainerExtension withLogging() {
         return this;
     }
 
-    public Neo4jContainerExtension withDebugger() {
```

Comment:
On line 136 of TestContainerUtil.java, this is commented out with a comment saying attach debugger. I wonder if perhaps it was useful there for debugging. If not, can you remove that line too :)